binding/gen.go

```diff
@@ -0,0 +1,126 @@
+// +build ignore
+
+package main
+
+import (
+	"os"
+	"path"
+	"runtime"
+	"text/template"
+
+	"fyne.io/fyne"
+)
+
+const headerTemplate = `// auto-generated
+// **** THIS FILE IS AUTO-GENERATED, PLEASE DO NOT EDIT IT **** //
+
+package binding
+
+import (
+{{ range . }}	"{{ . }}"
+{{ end }})
+`
+const unitBindingTemplate = `
+type {{ .Name }}Binding struct {
+	BaseBinding
+	Value {{ .Type }}
+}
+
+func (b *{{ .Name }}Binding) Get{{ .Name }}() {{ .Type }} {
+	return b.Value
+}
+
+func (b *{{ .Name }}Binding) Set(value interface{}) {
```

Comment:
Is it possible that this line was typed, avoiding the varying named methods below?